src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -85,67 +86,85 @@ const ResearchProfile = () => {
   ) : !person ? (
     <NotFound />
   ) : (
-    <BackgroundDiv>
-      <Helmet>
-        <title>{fullName}</title>
-      </Helmet>
-      <Typography variant="h1" sx={{ mt: '1rem', mb: '2rem' }}>
-        {fullName}
-      </Typography>
-      <Typography variant="h2">{t('common.employments')}</Typography>
-      {activeAffiliations.length > 0 ? (
-        <Box sx={lineSx}>
-          <WorkIcon />
-          <Box sx={textContainerSx}>
-            {activeAffiliations.map(({ organization }) => (
-              <AffiliationHierarchy key={organization} unitUri={organization} commaSeparated />
-            ))}
+    <Box>
+      <Box sx={{ bgcolor: 'person.main', py: '1.1rem', display: 'flex', alignItems: 'center', gap: '1rem' }}>
+        <Typography variant="h1" sx={{ ml: '2rem' }}>
+          {fullName}
+        </Typography>
+        {orcidUri && <img src={orcidIcon} height="20" alt="orcid" />}
+      </Box>
+      <BackgroundDiv>
+        <Helmet>
+          <title>{fullName}</title>
+        </Helmet>
+
+        {activeAffiliations.length > 0 ? (
+          <Box sx={lineSx}>
```

Comment:
`lineSx` er vel fra gammelt av, så ikke sikkert den trengs lenger for denne nye visningen?
```suggestion
```